### PR TITLE
added general app with cookies page

### PR DESCRIPTION
**Pull request by mike-allaway**
This MR adds the general app with the cookies page and necessary changes to 'core' (e.g. adding app to settings.py and to urls.py)

### CODE REVIEW

**Comment by mike-allaway** [comment]
This is the diff vs the template project:


```
Only in general: __pycache__
Only in general_template/templates/general: coming-soon.html
diff -ur general/templates/general/cookies.html general_template/templates/general/cookies.html
--- general/templates/general/cookies.html	2020-11-14 16:31:03.000000000 +0000
+++ general_template/templates/general/cookies.html	2020-11-23 14:18:46.000000000 +0000
@@ -1,85 +1,7 @@
-{% extends "base.html" %}
+{% extends 'base.html' %}
 {% load static %}
 
-{% block main %}
-    <div class="container">
-        <div>
-            <h2>Cookies information</h2>
-            <p>
-                For information about cookies, including what they are and how you can manage them, please visit the University of Birmingham's <a href="https://www.birmingham.ac.uk/privacy/cookies.aspx">cookies policy page</a>.
-            </p>
-        </div>
-        <div>
-            <h2>Cookies used on this website</h2>
-
-            <table>
-                <tr>
-                    <th>Provider</th>
-                    <th>Cookie</th>
-                    <th>Purpose</th>
-                    <th>Expiry</th>
-                </tr>
-                <tr>
-                    <td>Google Analytics</td>
-                    <td>_ga</td>
-                    <td>
-                        This helps us count how many people visit our website by tracking if you’ve visited before 
-                    </td>
-                    <td>
-                        2 years
-                    </td>
-                </tr>
-                <tr>
-                    <td>Google Analytics</td>
-                    <td>_gid</td>
-                    <td>
-                        This helps us count how many people visit our website by tracking if you’ve visited before 
-                    </td>
-                    <td>
-                        24 hours
-                    </td>
-                </tr>
-                <tr>
-                    <td>Google Analytics</td>
-                    <td>_gat</td>
-                    <td>
-                        Used to manage the rate at which page view requests are made 
-                    </td>
-                    <td>
-                        10 minutes
-                    </td>
-                </tr>
-                <tr>
-                    <td>This website</td>
-                    <td>cookie_msg_hide</td>
-                    <td>
-                        Used to know if you've previously hidden the cookies information messages on our site
-                    </td>
-                    <td>
-                        1000 days
-                    </td>
-                </tr>
-                <tr>
-                    <td>This website</td>
-                    <td>csrftoken</td>
-                    <td>
-                        A token used to enhance the security of the site by preventing Cross-Site Request Forgery (CSRF) attacks
-                    </td>
-                    <td>
-                        1 year
-                    </td>
-                </tr>
-                <tr>
-                    <td>This website</td>
-                    <td>sessionid</td>
-                    <td>
-                        Used to maintain your session when you're logged into your account on the website
-                    </td>
-                    <td>
-                        2 weeks
-                    </td>
-                </tr>
-            </table>
-        </div>
-    </div>
-{% endblock %}
\ No newline at end of file
+{% block content %}
+<h1>Cookies</h1>
+<p>The RSG Django Template website does not use cookies. For more information about cookies, including what they are and how you can manage them, please visit the University of Birmingham’s <a href="https://www.birmingham.ac.uk/privacy/cookies.aspx">cookies policy page</a>.</p>
+{% endblock %}
Only in general: tests.py
diff -ur general/urls.py general_template/urls.py
--- general/urls.py	2020-09-17 15:50:36.000000000 +0100
+++ general_template/urls.py	2020-11-23 14:18:46.000000000 +0000
@@ -1,6 +1,8 @@
 from django.urls import path
-from . import views
+from django.views.generic import TemplateView
 
+app_name = 'general'
 urlpatterns = [
-    path('cookies/', views.CookiesTemplateView.as_view(), name='cookies'),
+    path('', TemplateView.as_view(template_name="general/coming-soon.html"), name='comingsoon'),
+    path('cookies', TemplateView.as_view(template_name="general/cookies.html"), name='cookies'),
 ]
diff -ur general/views.py general_template/views.py
--- general/views.py	2020-09-17 15:50:36.000000000 +0100
+++ general_template/views.py	2020-11-23 14:18:46.000000000 +0000
@@ -1,8 +1,3 @@
-from django.views.generic import TemplateView
-
-
-class CookiesTemplateView(TemplateView):
-    """
-    Class-based view to show the cookies template
-    """
-    template_name = 'general/cookies.html'
+# This app only includes basic TemplateView generic views.
+# These are included in urls.py, within the urlpatterns array.
+# This is why this views.py file is empty.
```

**Comment by mike-allaway** [comment]
I just noticed the above diff is in reverse... woops. For sake of time can you just reverse it back when reading, but if you need me to redo it let me know

**Review by branfosj — DISMISSED** [review]
👍

File: django/general/templates/general/cookies.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+            <table>
+                <tr>
+                    <th>Provider</th>
+                    <th>Cookie</th>
+                    <th>Purpose</th>
+                    <th>Expiry</th>
+                </tr>
+                <tr>
+                    <td>Google Analytics</td>
```
Comment (branfosj) [inline django/general/templates/general/cookies.html:23]: Is this site have Google Analytics? If not then remove the cookie related to it.

Comment (mike-allaway) [reply django/general/templates/general/cookies.html:23]: it will, yeah, I'll set it up in a separate MR

**Review by branfosj — APPROVED** [review]
:)

**Pull request merged**